src/__tests__/hooks/medium.useEventOperations.spec.tsx

```diff
@@ -0,0 +1,255 @@
+import * as fetchJsonModule from '@apis/fetchJson.ts';
+import { ChakraProvider, useToast, UseToastOptions } from '@chakra-ui/react';
+import { useCreateEvent } from '@hooks/events/useCreateEvent.ts';
+import { useDeleteEvent } from '@hooks/events/useDeleteEvent.ts';
+import { useFetchEvents } from '@hooks/events/useFetchEvents.ts';
+import { useUpdateEvent } from '@hooks/events/useUpdateEvent.ts';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { CalendarManager } from '@templates/CalenderManager.tsx';
+import { act, renderHook, waitFor } from '@testing-library/react';
+import React from 'react';
+
+import { createTestQueryClient } from '../../__mocks__/createTestQueryClient.tsx';
+import { createEvent } from '../../__mocks__/Factory.ts';
+import { events } from '../../__mocks__/response/events.json' assert { type: 'json' };
+import { Event, EventForm } from '../../types.ts';
+
+const MOCK_EVENTS = [...events] as const;
+
+let toastCalls: any[] = [];
+
+// toast mock
+vi.mock('@chakra-ui/react', async () => {
+  const actual = await vi.importActual<typeof import('@chakra-ui/react')>('@chakra-ui/react');
+  return {
+    ...actual,
+    useToast: vi.fn().mockImplementation(() => (options: UseToastOptions) => {
+      toastCalls.push(options);
+    }),
+  };
+});
+
+const createWrapper =
+  (queryClient: QueryClient) =>
+  ({ children }: { children: React.ReactNode }) => {
+    return (
+      <QueryClientProvider client={queryClient}>
+        <ChakraProvider>
+          {children}
+          <CalendarManager />
+        </ChakraProvider>
+      </QueryClientProvider>
+    );
+  };
+
+describe('초기값 세팅', () => {
+  let queryEvent: {
+    current: {
+      fetchEvents: ReturnType<typeof useFetchEvents>;
+      updateEvent: ReturnType<typeof useUpdateEvent>;
+      createEvent: ReturnType<typeof useCreateEvent>;
+      deleteEvent: ReturnType<typeof useDeleteEvent>;
+    };
+  };
+
+  const toast = useToast();
+  const queryClient = createTestQueryClient(toast);
+  const wrapper = createWrapper(queryClient);
+
+  beforeEach(async () => {
+    // GIVEN: 각 테스트마다 초기 상태가 반영된 핸들러 설정
+    toastCalls = [];
+    queryClient.clear();
+
+    // GIVEN: 초기상태 불러오기
+    queryEvent = renderHook(
+      () => ({
+        fetchEvents: useFetchEvents(),
+        createEvent: useCreateEvent(),
+        updateEvent: useUpdateEvent(),
+        deleteEvent: useDeleteEvent(),
+      }),
+      { wrapper }
+    ).result;
```

Comment:
p1;
useEventOperation 테스트인데 따로따로 호출하고 계시네요!
useEventOperation를 호출하시는게 맞을듯합니다~
```suggestion
queryEvent = renderHook(() => useEventOperations(false), { wrapper }).result;
```